students/BrandonHenson/Lesson9/Mailroom_5.py

```diff
@@ -0,0 +1,138 @@
+# Brandon Henson
+# 5/17/18
+# Lesson 09 mailroom_5.py
+# Refactoring
+
+
+class Donor(object):
+
+    donor_history = {'Brandon Henson': [1005.49, 3116.72, 5200],
+                        'Alicia Henson': [21.47, 1500],
+                        'Michael Green': [2400.54],
+                        'Brandon Henson Jr': [355.42, 579.31],
+                        'Kaiya Henson': [636.9, 850.13, 125.23]}
```

Comment:
In the assignment instructions, it is recommended to have a `Donor` class for logic and data related to a single donor, and a separate class for the collection of donors, this will give you a better separation of concerns and allow your code to be more encapsulated, flexible and extensible.